atom/common/lib/asar.coffee

```diff
@@ -53,11 +53,20 @@ asarStatsToFsStats = (stats) ->
   }
 
 # Create a ENOENT error.
-createNotFoundError = (asarPath, filePath) ->
+createNotFoundError = (asarPath, filePath, callback) ->
```

Comment:
It sounds reasonable to me, we should probably rename it to `notFoundError` then, I want it to has a consistent name with `invalidArchiveError`.